### PR TITLE
Security

### PR DESCRIPTION
**Pull request by cyborgsimon**
default is false
if true
- the server/security directory is created
- for each generated collection a default
server/security/<collectionname>.coffee file is created
- add tests for the file creation and package

### CODE REVIEW

**Comment by cmalven** [comment]
Hey @cyborgsimon I appreciate you contributing to the project and you've contributed a lot in a short time, but I'm not interested in adding features that I don't use and I'm not confident the majority of people will use. I want to keep this boilerplate open-ended enough that it provides a solid base and set of tools and people can add the things they need.

The changes in this PR fall outside of that for me, and increase the complexity of the project without being useful for the majority of its users.

File: collection/templates/_collection.coffee [hunk 1]
```diff
@@ -1,3 +1 @@
-root = exports ? this
-
-root.<%= name.charAt(0).toUpperCase() + name.slice(1).toLowerCase() %> = new Meteor.Collection('<%= name.toLowerCase() %>')
\ No newline at end of file
+@<%= name.charAt(0).toUpperCase() + name.slice(1).toLowerCase() %> = new Meteor.Collection('<%= name.toLowerCase() %>')
```
Comment (cyborgsimon) [inline collection/templates/_collection.coffee]: I found it more convenient to make the collection global like this, if you do not like it i can revert this change.

**Comment by cyborgsimon** [comment]
I see.
If you do not mind I will fork and rename your project, to create and publish a more complex generator, fitting my needs on NPM.

**Comment by cmalven** [comment]
Of course I don't mind, and I appreciate you contributing to the project!

**Pull request closed without merging**